docs/documentation/release_notes/topics/26_0_0.adoc

```diff
@@ -130,6 +130,14 @@ not recommended at all in production deployments of Keycloak, it is fairly frequ
 of `localhost`. As an alternative to the `_LEGACY` cookies Keycloak now doesn't set the `secure` flag and sets `SameSite=Lax`
 instead of `SameSite=None` when it detects an insecure context is used.
 
+= Hostname v1 feature removed
+
+The deprecated hostname v1 feature was removed. This feature was deprecated in {project_name} 25 and replaced by hostname v2. If you are still using this feature, you must migrate to hostname v2. For more details, see the https://www.keycloak.org/server/hostname[Configuring the hostname (v2)].
```

Comment:
```suggestion
The deprecated `hostname:v1` feature was removed. This feature was deprecated in {project_name} 25 and replaced by `hostname:v2`. If you are still using this feature, you must migrate to hostname v2 options. For more details, see the https://www.keycloak.org/server/hostname[Configuring the hostname (v2)].
```

I'd like to distinguish the hostname v2 **feature** and **options**. IMHO, when we are talking about features, we implicitly mean Keycloak features, and it's natural for me to use the same syntax as for specifying the Keycloak features (`--features=hostname:v2`).

@shawkins I'd prefer this if it's ok for you. WDYT?

Reply:
> @shawkins I'd prefer this if it's ok for you. WDYT?

It seems like a follow-up issue would be needed to establish a convention and update the other feature references. 

cc @vmuzikar

Reply:
Follow-up works for me.

Reply:
To be fair, in docs we [reference](https://www.keycloak.org/server/features#_deprecated_features) it as `hostname-v1`. 😄 Not sure that's correct but it's a follow-up in any case.

Reply:
Unfortunately there are the following:
- feature label / description
- enum name: name_vX
- feature key: name-vX
- option key: name:vX
- properties file key (as it's used in testing): keycloak.profile.feature.name_vX (the later part should match the enum name)
- and other places use hostname vX